diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 1b51a5a1a..ca6bfd14b 100755
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -169,6 +169,9 @@ message(STATUS "boost: ${Boost_VERSION}")
 find_package(Python3 COMPONENTS Development REQUIRED)
 option(BUILD_PYTHON "Build the Python3 interface" ON)
 
+option(BUILD_MPL2 "Build MPL2 macro placer" ON)
+option(BUILD_PAR "Build PAR partitioner" ON)
+
 option(ALLOW_WARNINGS "Flag to allow compilation with compiler warnings: on by default" ON)
 if (NOT ALLOW_WARNINGS)
   add_compile_options(
@@ -251,10 +254,14 @@ add_subdirectory(cts)
 add_subdirectory(grt)
 add_subdirectory(tap)
 add_subdirectory(mpl)
-add_subdirectory(mpl2)
+if ((NOT ${CMAKE_SYSTEM_NAME} MATCHES "Darwin") AND BUILD_MPL2)
+  add_subdirectory(mpl2)
+endif()
 add_subdirectory(rcx)
 add_subdirectory(psm)
-add_subdirectory(par)
+if ((NOT ${CMAKE_SYSTEM_NAME} MATCHES "Darwin") AND BUILD_PAR)
+  add_subdirectory(par)
+endif()
 add_subdirectory(ant)
 add_subdirectory(gui)
 add_subdirectory(drt)
@@ -329,15 +336,23 @@ target_link_libraries(openroad
   ${CMAKE_THREAD_LIBS_INIT}
 )
 
-if (NOT ${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
+if ((NOT ${CMAKE_SYSTEM_NAME} MATCHES "Darwin") AND BUILD_MPL2)
   # mpl2 aborts with link error on darwin and par causes abseil link error at startup on apple silicon so do not link it.
-  target_link_libraries(openroad mpl2 par)
+  target_link_libraries(openroad mpl2)
   target_compile_definitions(openroad PRIVATE ENABLE_MPL2)
+else()
+  message(STATUS "Removing MPL2 to avoid run time fatal error.")
+endif()
+
+if ((NOT ${CMAKE_SYSTEM_NAME} MATCHES "Darwin") AND BUILD_PAR)
+  # mpl2 aborts with link error on darwin and par causes abseil link error at startup on apple silicon so do not link it.
+  target_link_libraries(openroad par)
   target_compile_definitions(openroad PRIVATE ENABLE_PAR)
 else()
-  message(STATUS "Removing MPL2 and PAR to avoid run time fatal error.")
+  message(STATUS "Removing PAR to avoid run time fatal error.")
 endif()
 
+
 # tclReadline
 if (TCL_READLINE_LIBRARY AND TCL_READLINE_H)
   target_compile_definitions(openroad PRIVATE ENABLE_READLINE)
@@ -407,7 +422,7 @@ if (Python3_FOUND AND BUILD_PYTHON)
     dft_py
   )
 
-  if (NOT ${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
+  if ((NOT ${CMAKE_SYSTEM_NAME} MATCHES "Darwin") AND BUILD_PAR)
     # par_py causes abseil link error at startup on apple silicon so do not link it.
     target_link_libraries(openroad par_py)
   else()
