diff --git a/src/gpl/CMakeLists.txt b/src/gpl/CMakeLists.txt
index d80f6778e..d997246d1 100644
--- a/src/gpl/CMakeLists.txt
+++ b/src/gpl/CMakeLists.txt
@@ -41,7 +41,6 @@ include("openroad")
 set(THREADS_PREFER_PTHREAD_FLAG ON)
 
 find_package(Eigen3 REQUIRED)
-find_package(ortools REQUIRED)
 find_package(LEMON NAMES LEMON lemon REQUIRED)
 find_package(OpenMP REQUIRED)
 
@@ -66,7 +65,6 @@ target_sources(gpl
     src/timingBase.cpp
     src/graphics.cpp
     src/solver.cpp
-    src/mbff.cpp
 )
 
 messages(TARGET gpl)
@@ -86,7 +84,6 @@ target_link_libraries(gpl
     OpenSTA
     rsz
     grt
-    ortools::ortools
     Threads::Threads
     OpenMP::OpenMP_CXX
 )
diff --git a/src/gpl/README.md b/src/gpl/README.md
index f391d2e67..db4f585b2 100644
--- a/src/gpl/README.md
+++ b/src/gpl/README.md
@@ -127,16 +127,6 @@ global_placement_debug -pause -update -inst -draw_bins -initial
 get_global_placement_uniform_density -pad_left -pad_right 
 ```
 
-### Cluster Flops
-
-Cluster single bit flops into multi-bit flops.
-
-```
-cluster_flops
-    [-tray_weight tray_weight]
-    [-timing_weight timing_weight]
-    [-max_split_size max_split_size]
-```
 
 #### General Arguments
 
diff --git a/src/gpl/include/gpl/Replace.h b/src/gpl/include/gpl/Replace.h
index ecf4c243b..ea28dbb68 100644
--- a/src/gpl/include/gpl/Replace.h
+++ b/src/gpl/include/gpl/Replace.h
@@ -84,7 +84,6 @@ class Replace
 
   void doIncrementalPlace();
   void doInitialPlace();
-  void runMBFF(int max_sz, float alpha, float beta, int threads);
 
   int doNesterovPlace(int start_iter = 0);
 
diff --git a/src/gpl/src/replace.cpp b/src/gpl/src/replace.cpp
index 3dc353f63..a8d3a6364 100644
--- a/src/gpl/src/replace.cpp
+++ b/src/gpl/src/replace.cpp
@@ -251,12 +251,6 @@ void Replace::doInitialPlace()
   ip_->doBicgstabPlace();
 }
 
-void Replace::runMBFF(int max_sz, float alpha, float beta, int threads)
-{
-  MBFF pntset(db_, sta_, log_, threads, 4, 10, gui_debug_);
-  pntset.Run(max_sz, alpha, beta);
-}
-
 bool Replace::initNesterovPlace()
 {
   if (!pbc_) {
diff --git a/src/gpl/src/replace.i b/src/gpl/src/replace.i
index 4ecb0a32c..abf18d15c 100644
--- a/src/gpl/src/replace.i
+++ b/src/gpl/src/replace.i
@@ -43,16 +43,6 @@ replace_nesterov_place_cmd()
   replace->doNesterovPlace();
 }
 
-
-void
-replace_run_mbff_cmd(int max_sz, float alpha, float beta) 
-{
-  Replace* replace = getReplace();
-  int threads = ord::OpenRoad::openRoad()->getThreadCount();
-  replace->runMBFF(max_sz, alpha, beta, threads);   
-}
-
-
 void
 set_density_cmd(float density)
 {
diff --git a/src/gpl/src/replace.tcl b/src/gpl/src/replace.tcl
index 199f70578..b8cf5f625 100644
--- a/src/gpl/src/replace.tcl
+++ b/src/gpl/src/replace.tcl
@@ -319,37 +319,6 @@ proc global_placement { args } {
 }
 
 
-sta::define_cmd_args "cluster_flops" {\
-    [-tray_weight tray_weight]\
-    [-timing_weight timing_weight]\
-    [-max_split_size max_split_size]\
-}
-
-proc cluster_flops { args } {
-  sta::parse_key_args "cluster_flops" args \
-    keys { -tray_weight -timing_weight -max_split_size }
-
-
-  set tray_weight 20.0
-  set timing_weight 1.0
-  set max_split_size -1
-
-  if { [info exists keys(-tray_weight)] } {
-    set tray_weight $keys(-tray_weight)
-  }
-
-  if { [info exists keys(-timing_weight)] } {
-    set timing_weight $keys(-timing_weight)
-  }
-
-  if { [info exists keys(-max_split_size)] } {
-    set max_split_size $keys(-max_split_size)
-  }
-
-  gpl::replace_run_mbff_cmd $max_split_size $tray_weight $timing_weight
-}
-
-
 namespace eval gpl {
 proc global_placement_debug { args } {
   sta::parse_key_args "global_placement_debug" args \
