diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 3383cc173..79e66cecb 100755
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -168,6 +168,8 @@ message(STATUS "boost: ${Boost_VERSION}")
 find_package(Python3 COMPONENTS Development REQUIRED)
 option(BUILD_PYTHON "Build the Python3 interface" ON)
 
+option(BUILD_MPL2 "Build MPL2 macro placer" ON)
+
 option(ALLOW_WARNINGS "Flag to allow compilation with compiler warnings: on by default" ON)
 if (NOT ALLOW_WARNINGS)
   add_compile_options(
@@ -250,7 +252,9 @@ add_subdirectory(cts)
 add_subdirectory(grt)
 add_subdirectory(tap)
 add_subdirectory(mpl)
-add_subdirectory(mpl2)
+if ((NOT ${CMAKE_SYSTEM_NAME} MATCHES "Darwin") AND BUILD_MPL2)
+  add_subdirectory(mpl2)
+endif()
 add_subdirectory(rcx)
 add_subdirectory(psm)
 add_subdirectory(par)
@@ -325,12 +329,12 @@ target_link_libraries(openroad
   ${CMAKE_THREAD_LIBS_INIT}
 )
 
-if (NOT ${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
+if ((NOT ${CMAKE_SYSTEM_NAME} MATCHES "Darwin") AND BUILD_MPL2)
   # mpl2 aborts with link error on darwin so do not link it.
   target_link_libraries(openroad mpl2)
   target_compile_definitions(openroad PRIVATE ENABLE_MPL2)
 else()
-  message(STATUS "Removing MPL2 to avoid run time fatal error.")
+  message(STATUS "MPL2 macro placer disabled.")
 endif()
 
 # tclReadline
