diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 99752d275..74a1ef540 100755
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -244,7 +244,6 @@ add_subdirectory(cts)
 add_subdirectory(grt)
 add_subdirectory(tap)
 add_subdirectory(mpl)
-add_subdirectory(mpl2)
 add_subdirectory(rcx)
 add_subdirectory(psm)
 add_subdirectory(par)
@@ -309,7 +308,6 @@ target_link_libraries(openroad
   drt
   dst
   mpl
-  mpl2
   psm
   ant
   par
diff --git a/src/OpenRoad.cc b/src/OpenRoad.cc
index dcf1ae707..0f0341ddb 100644
--- a/src/OpenRoad.cc
+++ b/src/OpenRoad.cc
@@ -57,7 +57,6 @@
 #include "gui/MakeGui.h"
 #include "ifp//MakeInitFloorplan.hh"
 #include "mpl/MakeMacroPlacer.h"
-#include "mpl2/MakeMacroPlacer.h"
 #include "odb/cdl.h"
 #include "odb/db.h"
 #include "odb/defin.h"
@@ -160,7 +159,6 @@ OpenRoad::~OpenRoad()
   deleteTritonCts(tritonCts_);
   deleteTapcell(tapcell_);
   deleteMacroPlacer(macro_placer_);
-  deleteMacroPlacer2(macro_placer2_);
   deleteOpenRCX(extractor_);
   deleteTritonRoute(detailed_router_);
   deleteReplace(replace_);
@@ -213,7 +211,6 @@ void OpenRoad::init(Tcl_Interp* tcl_interp)
   tritonCts_ = makeTritonCts();
   tapcell_ = makeTapcell();
   macro_placer_ = makeMacroPlacer();
-  macro_placer2_ = makeMacroPlacer2();
   extractor_ = makeOpenRCX();
   detailed_router_ = makeTritonRoute();
   replace_ = makeReplace();
@@ -247,7 +244,6 @@ void OpenRoad::init(Tcl_Interp* tcl_interp)
   initTritonCts(this);
   initTapcell(this);
   initMacroPlacer(this);
-  initMacroPlacer2(this);
   initOpenRCX(this);
   initPad(this);
   initRestructure(this);
