diff --git a/src/drt/CMakeLists.txt b/src/drt/CMakeLists.txt
index ce93a063e..2c3afc754 100644
--- a/src/drt/CMakeLists.txt
+++ b/src/drt/CMakeLists.txt
@@ -159,43 +159,6 @@ target_compile_options(drt
     $<$<CXX_COMPILER_ID:Clang>:-Wall -pedantic -Wcast-qual -Wredundant-decls -Wformat-security -Wno-gnu-zero-variadic-macro-arguments>
   )
 
-############################################################
-# Unit testing
-############################################################
-enable_testing()
-
-add_executable(trTest
-  ${FLEXROUTE_HOME}/test/gcTest.cpp
-  ${FLEXROUTE_HOME}/test/fixture.cpp
-)
-
-target_include_directories(trTest
-  PRIVATE
-  ${FLEXROUTE_HOME}/src
-  ${OPENROAD_HOME}/include
-)
-
-target_link_libraries(trTest
-  drt
-  odb
-)
-
-# Use the shared library if found.  We need to pass this info to
-# the code to select the corresponding include.  Using the shared
-# library speeds up compilation.
-if (Boost_unit_test_framework_FOUND)
-  message(STATUS "Boost unit_test_framework library found")
-  target_link_libraries(trTest
-    Boost::unit_test_framework
-  )
-  target_compile_definitions(trTest
-    PRIVATE
-    HAS_BOOST_UNIT_TEST_LIBRARY
-  )
-endif()
-
-add_test(NAME trTest COMMAND trTest)
-
 ############################################################
 # VTune ITT API
 ############################################################
diff --git a/src/odb/CMakeLists.txt b/src/odb/CMakeLists.txt
index 7e55b669c..4614df8bf 100644
--- a/src/odb/CMakeLists.txt
+++ b/src/odb/CMakeLists.txt
@@ -29,7 +29,6 @@ add_subdirectory(src/zutil)
 add_subdirectory(src/zlib)
 add_subdirectory(src/tm)
 add_subdirectory(src/cdl)
-add_subdirectory(test/cpp)
 ############################################################################
 ################################# SWIG #####################################
 ############################################################################
