diff --git a/tcl/board/numato_waxwing.cfg b/tcl/board/numato_waxwing.cfg
new file mode 100644
index 0000000..328e1e5
--- /dev/null
+++ tcl/board/numato_waxwing.cfg
@@ -0,0 +1,30 @@
+#
+# Numato Waxwing Spartan 6 FPGA Development Board
+#
+# https://numato.com/waxwing-spartan-6-fpga-development-board
+#
+# IMPORTANT:
+#   To use openocd with Waxwing using its onboard FTDI chip,
+#   put the jumpers P12 and P13 in following configuration:
+#
+#   P12: Jumpers one each between 2-4 and 3-5
+#   P13: Jumpers one each between 2-4 and 3-5
+#
+# NOTE:
+#   To put the jumpers configuration back to factory configuration (SPI flash connnections):
+#
+#   P12: Jumpers one each between 1-2 and 5-6
+#   P13: Jumpers one each between 1-2 and 5-6
+
+interface ftdi
+ftdi_device_desc "Waxwing Spartan 6 FPGA Board"
+ftdi_vid_pid 0x0403 0x6010
+# channel 1 is for custom purpose by users (like uart, fifo etc)
+ftdi_channel 0
+# just TCK TDI TDO TMS, no reset
+ftdi_layout_init 0x0008 0x000b
+reset_config none
+adapter_khz 10000
+
+source [find cpld/xilinx-xc6s.cfg]
+source [find cpld/jtagspi.cfg]
