diff --git a/tcl/cpld/xilinx-xc7.cfg b/tcl/cpld/xilinx-xc7.cfg
index d5824f8..4c0502c 100644
--- a/tcl/cpld/xilinx-xc7.cfg
+++ b/tcl/cpld/xilinx-xc7.cfg
@@ -9,7 +9,15 @@
 
 # the 4 top bits (28:31) are the die stepping/revisions. ignore it.
 jtag newtap $_CHIPNAME tap -irlen 6 -ignore-version \
+	-expected-id 0x03622093 \
+	-expected-id 0x03620093 \
+	-expected-id 0x037C4093 \
+	-expected-id 0x0362F093 \
+	-expected-id 0x037C8093 \
+	-expected-id 0x037C7093 \
+	-expected-id 0x037C3093 \
 	-expected-id 0x0362E093 \
+	-expected-id 0x037C2093 \
 	-expected-id 0x0362D093 \
 	-expected-id 0x0362C093 \
 	-expected-id 0x03632093 \
