diff --git a/yosys/Makefile b/yosys/Makefile
index b5cfb6b42..6138db005 100644
--- a/yosys/Makefile
+++ b/yosys/Makefile
@@ -348,6 +348,20 @@ ABCMKARGS += ARCHFLAGS="-DABC_USE_STDINT_H -DWIN32_NO_DLL -DHAVE_STRUCT_TIMESPEC
 ABCMKARGS += LIBS="-lpthread -s" ABC_USE_NO_READLINE=0 CC="x86_64-w64-mingw32-gcc" CXX="$(CXX)"
 EXE = .exe
 
+else ifeq ($(CONFIG),conda-linux)
+CXX = $(notdir $(CC))
+LD = $(notdir $(CC))
+CXXFLAGS += -std=c++11 -Os -fno-merge-constants
+CFLAGS += -Wno-unused-function -Wno-unused-but-set-variable
+ABCMKARGS += "ABC_READLINE_INCLUDES=-I${PREFIX}/include"
+ABCMKARGS += "ABC_READLINE_LIBRARIES=-L${PREFIX}/lib -lreadline"
+
+else ifeq ($(CONFIG),conda-mac)
+CXX = $(notdir $(CC))
+LD = $(notdir $(CXX))
+CXXFLAGS += -std=c++11 -Os
+ABCMKARGS += ARCHFLAGS="-DABC_USE_STDINT_H"
+
 else ifneq ($(CONFIG),none)
 $(error Invalid CONFIG setting '$(CONFIG)'. Valid values: clang, gcc, gcc-4.8, emcc, mxe, msys2-32, msys2-64)
 endif
