--- Surelog/CMakeLists.txt
+++ Surelog/CMakeLists.txt
@@ -44,7 +44,7 @@
 find_package(SWIG 3.0 REQUIRED)
 
 # Python
-find_package(Python3 3.3 REQUIRED COMPONENTS Interpreter Development)
+find_package(Python3 REQUIRED COMPONENTS Interpreter Development)
 message(STATUS "Python3_LIBRARIES = ${Python3_LIBRARIES}")
 message(STATUS "Python3_EXECUTABLE = ${Python3_EXECUTABLE}")
 message(STATUS "Python3_INCLUDE_DIRS = ${Python3_INCLUDE_DIRS}")
