Submodule yosys-f4pga-plugins contains modified content
diff --git a/yosys-f4pga-plugins/Makefile_plugin.common b/yosys-f4pga-plugins/Makefile_plugin.common
index 45c2640..9b838e0 100644
--- a/yosys-f4pga-plugins/Makefile_plugin.common
+++ b/yosys-f4pga-plugins/Makefile_plugin.common
@@ -46,13 +46,13 @@ TOP_DIR := $(abspath $(dir $(lastword $(MAKEFILE_LIST))))
 
 _MAKEFILES := $(abspath $(filter-out %.d,$(MAKEFILE_LIST)))
 
-# Either find yosys in system and use its path or use the given path
-YOSYS_PATH ?= $(realpath $(dir $(shell command -v yosys))/..)
+# Either find uhdm-yosys in system and use its path or use the given path
+YOSYS_PATH ?= $(realpath $(dir $(shell which uhdm-yosys))/..)
 
-# Find yosys-config, throw an error if not found
-YOSYS_CONFIG = $(YOSYS_PATH)/bin/yosys-config
+# Find uhdm-yosys-config, throw an error if not found
+YOSYS_CONFIG = $(YOSYS_PATH)/bin/uhdm-yosys-config
 ifeq (,$(wildcard $(YOSYS_CONFIG)))
-  $(error "Didn't find 'yosys-config' under '$(YOSYS_PATH)'")
+  $(error "Didn't find 'uhdm-yosys-config' under '$(YOSYS_PATH)'")
 endif
 
 CXX ?= $(shell $(YOSYS_CONFIG) --cxx)
diff --git a/yosys-f4pga-plugins/Makefile_test.common b/yosys-f4pga-plugins/Makefile_test.common
index 5d72327..75096de 100644
--- a/yosys-f4pga-plugins/Makefile_test.common
+++ b/yosys-f4pga-plugins/Makefile_test.common
@@ -16,13 +16,13 @@
 
 SHELL := /usr/bin/env bash
 
-# Either find yosys in system and use its path or use the given path
-YOSYS_PATH ?= $(realpath $(dir $(shell command -v yosys))/..)
+# Either find uhdm-yosys in system and use its path or use the given path
+YOSYS_PATH ?= $(realpath $(dir $(shell which uhdm-yosys))/..)
 
-# Find yosys-config, throw an error if not found
-YOSYS_CONFIG = $(YOSYS_PATH)/bin/yosys-config
+# Find uhdm-yosys-config, throw an error if not found
+YOSYS_CONFIG = $(YOSYS_PATH)/bin/uhdm-yosys-config
 ifeq (,$(wildcard $(YOSYS_CONFIG)))
-$(error "Didn't find 'yosys-config' under '$(YOSYS_PATH)'")
+$(error "Didn't find 'uhdm-yosys-config' under '$(YOSYS_PATH)'")
 endif
 
 GTEST_DIR ?= $(abspath ../../third_party/googletest)
@@ -51,7 +51,7 @@ $(1)/ok: $(1)/$$(notdir $(1).v)
 	echo "source $(TEST_UTILS)" > run-$$(notdir $(1)).tcl ;\
 	echo "source $$(notdir $(1)).tcl" >> run-$$(notdir $(1)).tcl ;\
 	DESIGN_TOP=$$(notdir $(1)) TEST_OUTPUT_PREFIX=./ \
-	yosys -c "run-$$(notdir $(1)).tcl" -q -q -l $$(notdir $(1)).log; \
+	uhdm-yosys -c "run-$$(notdir $(1)).tcl" -q -q -l $$(notdir $(1)).log; \
 	RETVAL=$$$$?; \
 	rm -f run-$$(notdir $(1)).tcl; \
 	if [ ! -z "$$($(1)_negative)" ] && [ $$($(1)_negative) -eq 1 ]; then \
