From e57dc500ef22047e2d57c1c4bcae83260f6d3672 Mon Sep 17 00:00:00 2001
From: Johan Euphrosine <proppy@google.com>
Date: Tue, 18 Oct 2022 22:29:37 +0900
Subject: [PATCH] flow: skip mounted version for local install

Signed-off-by: Johan Euphrosine <proppy@google.com>
---
 flow.tcl | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/flow.tcl b/flow.tcl
index 2ec16863a..6a9eec912 100755
--- a/flow.tcl
+++ b/flow.tcl
@@ -386,9 +386,11 @@ if {[catch {exec cat $::env(OPENLANE_ROOT)/install/installed_version} ::env(OPEN
     }
 }
 
-if {![catch {exec git --git-dir $::env(OPENLANE_ROOT)/.git rev-parse HEAD} ::env(OPENLANE_MOUNTED_SCRIPTS_VERSION)]} {
-    if { $::env(OPENLANE_VERSION) == $::env(OPENLANE_MOUNTED_SCRIPTS_VERSION)} {
-        unset ::env(OPENLANE_MOUNTED_SCRIPTS_VERSION)
+if { ! [info exists ::env(OPENLANE_LOCAL_INSTALL)] || ! $::env(OPENLANE_LOCAL_INSTALL)} {
+    if {![catch {exec git --git-dir $::env(OPENLANE_ROOT)/.git rev-parse HEAD} ::env(OPENLANE_MOUNTED_SCRIPTS_VERSION)]} {
+        if { $::env(OPENLANE_VERSION) == $::env(OPENLANE_MOUNTED_SCRIPTS_VERSION)} {
+            unset ::env(OPENLANE_MOUNTED_SCRIPTS_VERSION)
+        }
     }
 }
 
