diff --git a/fpga_interchange/interchange_capnp.py b/fpga_interchange/interchange_capnp.py
index c902259..df6a472 100644
--- a/fpga_interchange/interchange_capnp.py
+++ b/fpga_interchange/interchange_capnp.py
@@ -835,6 +835,10 @@ class Interchange():
     def __init__(self, schema_directory):
 
         search_path = [os.path.dirname(os.path.dirname(capnp.__file__))]
+        if 'CONDA_BUILD' in os.environ:
+            search_path.append(
+                os.path.join(os.environ['PREFIX'], 'include'))
+
         if 'CONDA_PREFIX' in os.environ:
             search_path.append(
                 os.path.join(os.environ['CONDA_PREFIX'], 'include'))
