diff --git a/gf180mcu/Makefile.in b/gf180mcu/Makefile.in
index 5a71821..2842a03 100644
--- a/gf180mcu/Makefile.in
+++ b/gf180mcu/Makefile.in
@@ -718,7 +718,7 @@ klayout-%: ${GF180MCU_PV_PATH} ${GF180MCU_PR_PATH}
 
 	cp -rp ${GF180MCU_PV_PATH}/klayout/drc/* \
 		${KLAYOUT_STAGING_$*}/drc
-	cp -rp ${GF180MCU_PR_PATH}/rules/klayout/lvs/* \
+	cp -rp ${GF180MCU_PV_PATH}/klayout/lvs/* \
 		${KLAYOUT_STAGING_$*}/lvs
 	cp -rp ${GF180MCU_PR_PATH}/cells/klayout/pymacros/cells/* \
 		${KLAYOUT_STAGING_$*}/pymacros
